src/BioformatsLoader.jl

```diff
@@ -109,6 +155,60 @@ function bf_import(fname::String; stdout_redirect=STDOUT)
                 close(rd)
         end
         return images
+
+function metadata(fname::String; stdout_redirect=STDOUT)
+
+	rd, wr = redirect_stdout()
+
+	oxr = OMEXMLReader()
+	set_id(oxr, fname)
+	xml = parse_string(jcall(oxr.meta, "dumpXML", JString, ()))
+	properties = Dict{String, Any}()
+	metalst = get_elements_by_tagname(root(xml), "Image")
+
+	SizeC = jcall(oxr.reader, "getSizeC", jint, ())
+
+	for a = attributes(metalst[1])
+		push!(properties, "$(name(a))"=>value(a))
+	end
+	for a = attributes(find_element(metalst[1], "Pixels"))
+		try
+			push!(properties, "$(name(a))"=> parse(Float64, value(a)))
+		    catch ex
+		    	if ex isa ArgumentError
+		        	push!(properties, "$(name(a))"=>  value(a))
+		        end
+		 end
+ 	end
+
+	for i in 1:SizeC
+		for a in attributes(metalst[1]["Pixels"][1]["Channel"][i])
+			if name(a)== "EmissionWavelength"
+		    	push!(properties, "channel_$i"=>value(a))
+		    end
+		 end
+	 end
+
+	redirect_stdout(STDOUT)
+	@async begin
+		while !eof(rd)
+			write(stdout_redirect, read(rd))
+		end
+		close(rd)
+	end
+	return properties, metalst, xml
+end
+
+function starting(MEM::Int)
+
+	# download bioformats_package.jar at https://www.openmicroscopy.org/bio-formats/downloads/
+
+	bfpkg_path = "/jars/bioformats_package.jar"
```

Comment:
This is an absolute path, you would probably want something like `"./jars/bioformats_package.jar"` or rather `dirname(pathof(BioformatsLoader)) * "../jars/bioformats_package.jar"` but since `starting` isn't explicitly exported you can leave this here for the moment. As soon as this gets merged I will fix this the way [Images.jl](https://github.com/JuliaImages/Images.jl) does using [BinaryProvider.jl](https://github.com/JuliaPackaging/BinaryProvider.jl).